From a53923977d5d1f5e2381ae72c6ff49b0bbc9d7b1 Mon Sep 17 00:00:00 2001
From: Edwin Moquete <edwinmmoquete@gmail.com>
Date: Sun, 06 Aug 2023 02:09:44 +0000
Subject: [PATCH] sdm845: Label kernel 4.19 devfreq nodes

Change-Id: I4668508296abfe3caa9cf9752ae9e6a49e7b93fc
---

diff --git a/generic/vendor/sdm845/genfs_contexts b/generic/vendor/sdm845/genfs_contexts
index 5b75c0e..4a24696 100644
--- a/generic/vendor/sdm845/genfs_contexts
+++ b/generic/vendor/sdm845/genfs_contexts
@@ -31,6 +31,20 @@
 
 genfscon sysfs /devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8998@0:qcom,pm8998_rtc/rtc u:object_r:sysfs_rtc:s0
 
+# DCVS nodes
+genfscon sysfs /devices/platform/soc/17d41000.qcom,devfreq-l3/17d41000.qcom,devfreq-l3:qcom,cpu0-cpu-l3-lat/devfreq   u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/17d41000.qcom,devfreq-l3/17d41000.qcom,devfreq-l3:qcom,cpu4-cpu-l3-lat/devfreq   u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/17d41000.qcom,devfreq-l3/17d41000.qcom,devfreq-l3:qcom,cdsp-cdsp-l3-lat/devfreq  u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu-cpu-llcc-bw/devfreq          u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu-llcc-ddr-bw/devfreq          u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu0-cpu-l3-lat/devfreq          u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu0-cpu-llcc-lat/devfreq        u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu0-cpu-ddr-latfloor/devfreq    u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu4-cpu-ddr-latfloor/devfreq    u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu4-cpu-l3-lat/devfreq          u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cpu4-cpu-llcc-lat/devfreq        u:object_r:vendor_sysfs_devfreq:s0
+genfscon sysfs /devices/platform/soc/soc:qcom,cdsp-cdsp-l3-lat/devfreq         u:object_r:vendor_sysfs_devfreq:s0
+
 #qdss sysfs-node
 genfscon sysfs /devices/platform/soc/6047000.tmc/coresight-tmc-etf u:object_r:vendor_sysfs_qdss_dev:s0
 genfscon sysfs /devices/platform/soc/6048000.tmc/coresight-tmc-etr u:object_r:vendor_sysfs_qdss_dev:s0
